IHP/IDE/Data/Controller.hs

```diff
@@ -109,6 +111,18 @@ instance Controller DataController where
         let targetId = cs id
         PG.close connection
         render EditValueView { .. }
+
+    action ToggleBooleanFieldAction { tableName, targetName, id } = do
+        let id :: String = cs (param @Text "id")
+        let tableName = param "tableName"
+        connection <- connectToAppDb
+        tableNames <- fetchTableNames connection
+        tableCols <- fetchTableCols connection tableName
+        let query = "UPDATE " <> tableName <> " SET " <> targetName <> " = NOT " <> targetName <> " WHERE id = '" <> cs id <> "'"
+        putStrLn (query)
+        PG.execute_ connection (PG.Query . cs $! query)
```

Comment:
We could use `PG.execute` (without `_` at the end) and then use `?` in the query for substituion. Otherwise this code is not safe against sql injection.

```
let query = PG.Query (cs $! "UPDATE ? SET ? = NOT ? WHERE id = ?")
let params = (PG.Identifier tableName, PG.Identifier targetName, PG.Identifier targetName, id)
PG.execute connection query params
```